webapp/crashstats/api/views.py

```diff
@@ -476,22 +476,22 @@ def crash_verify(request):
 
     data = {"uuid": crash_id}
 
-    # Check S3 crash bucket for raw and processed crash data
+    # Check crash bucket for raw and processed crash data
     raw_api = models.RawCrash()
     try:
         raw_api.get(crash_id=crash_id, dont_cache=True, refresh_cache=True)
         has_raw_crash = True
     except CrashIDNotFound:
         has_raw_crash = False
-    data["s3_raw_crash"] = has_raw_crash
+    data["raw_crash"] = has_raw_crash
```

Comment:
i don't know how important changing these keys is, if it's important i'm happy to revert this change and worry about the s3 prefix being less than accurate later.